spec/features/publish/providers_index_spec.rb

```diff
@@ -47,6 +45,26 @@
     and_i_click_the_change_organisation_link
     and_i_click_on_a_provider
     i_should_be_on_the_courses_index_page_in_the_same_recruitment_cycle
+
+    # Tests below for a couple of bugs which prevented those pages from knowing about the recruitment cycle
+
+    when_i_click_on_users
+    i_should_see_the_recruitment_cycle_text
+
+    when_i_click_on_organisations
+    i_should_see_the_recruitment_cycle_text
+  end
+
+  def i_should_see_the_recruitment_cycle_text
+    expect(page).to have_text "- 2021 to 2022 - current"
+  end
+
+  def when_i_click_on_users
+    click_link "Users"
```

Comment:
Let's put these links in the page objects. They'll likely need to be re-used in other specs. Even if it goes in `spec/support/page_objects/publish/header.rb` or `spec/support/page_objects/publish/primary_nav.rb` that should be fine for now